src/vivarium_testing_utils/automated_validation/data_transformation/measures.py

```diff
@@ -144,48 +143,68 @@ class Incidence(RatioMeasure):
     """Computes Susceptible Population Incidence Rate."""
 
     def __init__(self, cause: str) -> None:
-        self.measure_key = f"cause.{cause}.incidence_rate"
+        self.measure_key = self.artifact_key = f"cause.{cause}.incidence_rate"
```

Comment:
Is there a reason cause can't go into the constructor of RatioMeasure? Then you would only have to define this in one class

Reply:
I also think you could probably have a dict or some sort of mapping which would allow you to pass the measure in and infer this.